For-Loop/07.VowelsSum/07.VowelsSum.cs

```diff
@@ -0,0 +1,47 @@
+﻿using System;
+
+namespace _07.VowelsSum
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            string text = Console.ReadLine();
+
+            const int valueOfLetterA = 1;
+            const int valueOfLetterE = 2;
+            const int valueOfLetterI = 3;
+            const int valueOfLetterO = 4;
+            const int valueOfLetterU = 5;
+
+
+            int vowelsSum = 0;
+
+            for (int i = 0; i < text.Length; i++)
+            {
+                if (text[i] == 'a')
```

Comment:
In this type of long if else chain, switch case is a lot more readable 
```suggestion
                switch(text[i])
                    case 'a': vowelsSum += valueOfLetterA; break;
                    case 'e': .....
```